torch/nn/functional.py

```diff
@@ -86,23 +88,24 @@ def conv1d(input, weight, bias=None, stride=1, padding=0, dilation=1,
 def conv3d(input, weight, bias=None, stride=1, padding=0, dilation=1,
            groups=1):
     """Applies a 3D convolution over an input image composed of several input
-        planes.
```

Comment:
This space seems to make [gray rectangle](http://pytorch.org/docs/master/nn.html#id18) in PyTorch docs.